src/AppInstallerSharedLib/AppInstallerLogging.cpp

```diff
@@ -97,6 +125,31 @@ namespace AppInstaller::Logging
         m_enabledChannels &= ~ConvertChannelToBitmask(channel);
     }
 
+    uint64_t DiagnosticLogger::ConvertChannelToBitmask(Channel channel)
```

Comment:
I think I actually didn't do it originally to prevent "combining" channels.  But the current state of the code outweighs trying to prevent that mistake structurally.